### DISCUSSION TITLE
Skill Tree REVAMP

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#2180] [Ideas]
G'Day!

Myself with help from others have been working on some ideas regarding new revamped skill trees to replace the ones we have now. 

Currently the skill trees are okay but lack a bit of complexity and not to mention there are skills that include features which have not been implemented or planned as of yet. 

My proposal would give the players more options and variety to how they want to play the game. Not only would it add variety and a bit of extra decision-making on behalf of the player, but it would give a big incentive for players to level up to gain more skill points.

At level 40 a player has pretty much covered most of the skills present/they wanted, whilst these new skill trees would drive the established/maxed out players to progress further and have something to aim for. 

Theoretically once these new trees are released everyone would be given a complete skill reset and points would be reallocated back to them. There is an ongoing discussion to the amount of skill points handed out at x levels so the points received back might be different... watch this space.

Release of the revamped skill trees also means that devs can increase the current max levels to 100-200+, established/hardcore players can spend their time levelling up and therefore have something to do/aim for and devs can freely focus on the new player experience, lvl 10-20-30+ islands, low level sfts/items etc.

New features/gameplay items could be added as a new skill if needed OR added into existing trees simply as a new branch. A new animal without any skills present could have a couple of branches divewrging from the tree.

Below are the proposed skill trees, **PLEASE NOTE: IT IS ALL A DRAFT, VALUES/SKILLS ARE NOT FINAL AND THINGS WILL CHANGE, SOME ARE UTTER GARBAGED OR OP AND HAVE ALREADY BEEN REPLACED**

Blue arrows = 1 requirement (the skill beforehand)
Black arrows = Multiple requirements (Must have all prior skills to unlock next)

![image](https://user-images.githubusercontent.com/108110065/216873567-d3a4eb33-717a-44a5-99df-6c26065e3b4a.png)
![image](https://user-images.githubusercontent.com/108110065/216873632-b7823b4c-48d9-435a-8e85-3ac9d13a248c.png)

There is also possibility to make the skill tree even more indepth, below is an example of such and how it would look on mobile. Last 2 ULTIMATE Skills need to be scrolled down to see:
![image](https://user-images.githubusercontent.com/108110065/216870979-dad0697e-7f48-419d-8380-98488e690d3c.png)

![image](https://user-images.githubusercontent.com/108110065/216869616-2d8e2e34-f930-48ff-9f9a-b8fd6c26f556.png)

**Some of the skills are a bit repetitive and some are missing entirely (wood tree), it is still WIP and needs a balance pass.
DO NOT assume the skills you see will be LIVE because half of them will probably not be (Although, i buffed buckaroo and money tree i hope they stay)** 

Feedback I would appreciate comments on but not exclusively:

- Is it too much? Would you rather just stick with the simplicity of the previous skill tree?
- Is the UI an issue? Does having to scroll down a bit to see the last tier of skills cause annoyances?
- Do you like coffee?

Gathering feedback before I dedicate more time in finishing this :)
Many thanks for reading this far :D

**Comment by veenhdek** [comment]
I like the added complexity! Makes it more strategic and keeps people engaged to aim for something over time.

UI is great. I have to scroll most things on mobile including the farm as a whole and my inventory, so feels natural!

Big tea guy myself, earl grey or chai are my favorite but can go for a cup o’ joe every once in a while.

**Comment by Kay1797** [comment]
I love the idea of adding complexity to the skill trees. 
With the current system it is too easy to get everything you want/need to buff your farm, and it requires very little decision making. 

Does this also include future resets, or just the 1 time reset at the implementation?

The idea of giving us more experienced players something like a lvl 100 to grind for could be great! as we will be burning resources, stay engaged, and as you mentioned can give the devs time to focus on the new player experience. 
All in All, i say this is worth developing.

**Reply by TheDcol** [reply]
Thanks for the feedback

I imagine it would be a complete one time reset at release, so if this gets implemented and launched, you get all your points back and can pick again. 

But i am not knowledgeable in the technical side of it

**Comment by SuperSynapse** [comment]
I think you have a lot of things going in this proposal. I really like it!

A few notes that might worthwhile:

1) A system where rather than having arrows, it's required to have a certain number of skills in each "Tier" before being able to move down the next tier. This would allow more lateral movement and the choice to move down the tier, or continue to get another skill in the tier you've already met the requirements for. Think Borderlands, League of Legends, or Deep Rock Galactic.

2) Potentially have "achievements" before unlocking certain skills. For example: To unlock "10% quicker chicken yield" you have to first farm 1,000 eggs.

Depending on the circumstance, this could also encourage cross play, for example you could have the requirement instead be "Harvest 1,000 wheat". Which would encourage a player to "stock up on wheat" before getting a certain boost to their chicken yield; or farm certain ingredients before upgrading their cooking, or chop a certain amount of wood before getting an upgrade to stone.

As a side benefit, this somewhat discourages going "all in" to stats like gold, and only buying iron/wood to meet that skill requirement, or only buying wheat after going "all in" on chickens. Requires the player to first earn the path to that new upgrade before getting it.

3) Somewhat to the point of number 2; It would be cool to to have some prerequisites to certain trees. For example:
- To unlock the Fruit Tree, you must be lvl 20
- To unlock the Animal tree, you must have a lvl 2 hen house or barn
- To unlock the rock tree, you must have gathered 100 gold

Base level skills (crops, wood, cooking) could be avaliable at lvl 1. But more advanced skills (Fruit, animals, minerals) would be locked behind other tasks.

At the end of the day, why should anyone go for animals, rocks or fruit first? And therein lies the good direction of pushing them into crops, wood, cooking first.

The complexity is really good IMO. It gives a straightforward direction
 - "Do I want to go for crops or wood?"
 And then leads the player into a deeper level:
 - "Do I want to go for 10% speed or 5% yield"
 I also like the furthering of that to:
 - "If I want to go animals, I need to prepare a few things first, such as farming wheat, or building henhouses" 
 and:
 - "if I want to go chickens I first need to do this, if I want to go cows, I first need to do that."
 
 
Currently the Achievements are for the most part unrewarding. It's nice to get a new bear, or 0.5 SFL, but at the end of the day it lacks the sense of advancement like your proposal. Which is what EVERYONE craves in an engaging game. Logging into the game week after week and seeing how you've progressed relative to when you started!

**Reply by TheDcol** [reply]
Thanks for the indepth feedback! 😁

Regarding number 1, this would be interesting to look at, could provide a separate diagram to compare

2 and 3, nice ideas to make the trees more intricate and tie them to the gameplay of the player. My only worry with setting up these kinds of prerequisites is if the players will feel more restricted than they already might feel, would be up to devs with how they want their game to feel. But would be interesting!

If you set up prerequisites such as your example of 'harvest 1,000 eggs to unlock x skill' you are kind of forcing the player to stick to what they are doing already and just unlock crop tree. The player might have 5-6 skill points just sitting there and now has to grind x and y to unlock something, might feel a bit weird to have skill points but not able to use them

I guess it comes down to how much freedom do players have and how much are they supposed to have, everyone will have an opinion on this 😅

I also agree achievements are very lacklustre and more could be done

Edit: setting prerequisites to unlock skills is a good way to guide players into what you want them to do, just worried if the gane wants players to do that overall. Technically there already is some guidance with the pathing of the tree

**Reply by paduv** [reply]
I also like the idea that you'd need to have completed achievements associated with the skills before you can choose them.
Achievements would play the role of specialized xp point (wood xp, crops xp etc), except more fun than just numbers.
I'd see it as one more tool for the devs to make the right balance between "extractors" and "investors": Certain achievements might require sacrifice or commitment from the player, and the skill associated be more powerful.
This articulation Skills-Achievements could be a real plus in the progression gameplay!

To open up the possibilities for the player, a skill might ask for one of several Achievements, not just the one & only per skill. So, for example you could get the first crop skill by having previously harvested 10,000 sunflowers, or 500 parsnips, or another achievement. Or it could ask for more than one etc

That said, it shouldn't constitute a barrier for new players, e.g. the first skills of each tree should stay 'free-of-achievements'.
But it would prevent relative newcomers to jump too fast to the highest skills.
Again, it's a matter of balance as to how these limits are set exactly.

Another point with the skill-achievement articulation: it will restrain players from resetting their skills every two days (once reset is available), immediately changing their whole configuration: now they'll first have to grind the achievements for their new choice of skills.

btw, i like the new skills tree, & i'm all for a bit more complexity :)

**Comment by adamhannigan** [comment]
Always impressive the work you do! @TheDcol 

I will let the community run the bulk of this discussion - feel free to update and make decisions. 

Right now I will pin this to get more exposure - tomorrow we can do an announcement in Discord if we need broader feedback 💪

**Comment by Polysemouse** [comment]
Over all I really like the new idea!  The added complexity of the skill trees keeps players engaged and allow them to strategize on their playing style.

The level curve can be readjusted to include more levels in between (eg. old level 5 will be the new level 10, old level 30 will be the new level 60 etc.).  That way we can have more skill points to spend for all these new skills.  Alternatively (or additionally), each level can reward players with multiple skill points (eg. 3-5 skillpoints per level, however each skill would need 5-10 skillpoints to be claimed.

Is it too much? Would you rather just stick with the simplicity of the previous skill tree?
- Looks like the right balance.  I would probably start with 1 branch for each skill tree before it branches off to multiple ones (and ultimately merge back in for some branches.  I would also keep multiple branches at the end instead of 1 so it feels like a tree

Is the UI an issue? Does having to scroll down a bit to see the last tier of skills cause annoyances?
- It could be an issue if there are 5 or more skill in the same tree level for smaller screens, so this would need to be kept in mind while designing vertical skill trees.  Otherwise time would need to spend on UI/UX improvements to see which ones makes the most sense

Do you like coffee?
- Yes, when coffee beans in Sunflower Land?

**Comment by LeShykun** [comment]
- Is it too much? Would you rather just stick with the simplicity of the previous skill tree?

○ Yes! Too many skills out there are really good and give more motivation to actually level up your Bumpkin even further!

- Is the UI an issue? Does having to scroll down a bit to see the last tier of skills cause annoyances?

○ Honestly I'm really fine with this UI, maybe can add a button to zoom out and see whole skills idk, Probably no (at least for me)

- Do you like coffee?

Nope

**Comment by esp123** [comment]
love this idea, this will open a lot of strategy and will force player to prioritize certain skills and really make a bumpkin more unique than others!! LFG!

**Reply by TheDcol** [reply]
Thanks that is what I would like to see in the game 😁

**Comment by Xeko-92** [comment]
Lazy Farmer after Golden Sunflower doesn't make much sense to me. I doubt many people would be taking both of those skills.

Also get rid of the .1 SFL drop from trees, or increase it, because it's actually worse for some players than just getting wood.

I don't like the expansions cost less skill, it's a pretty weak effect compared to the final skill in a lot of the other trees.

**Reply by TheDcol** [reply]
Thx for the comments, as previously stated in the post the skills are nowhere near final and should only be looked at as a guide for now, sitting down tomorrow to consider suggestions and have a proper think, looking at the whole gameplay of sfl

Reduced expansion costs could be really important down the line... Would be less useful for current players who expanded max capacity, but when there will be 10-50+ expansions coming it could be good i think. Maybe even bumping it to 15%-20% less costs could be interesting

Will be posting an updated one soon with official proposed skills, this was a very basic mock 😁

**Comment by docedeamendoim22** [comment]
great idea, this really helps to make each land unique

**Comment by DeccoMx** [comment]
- Is it too much? Would you rather just stick with the simplicity of the previous skill tree?
I Really love the proposal, and don't i don't think it's too much... Or well, maybe it's too much but i love too much, i love the extra complexity that this could add to the game experience. 

- Is the UI an issue? Does having to scroll down a bit to see the last tier of skills cause annoyances?
It will annoy me, but only on the cellphone, and i could live with it. 

- Do you like coffee?
I love it

**Comment by ShiningMinions** [comment]
Is it too much? Would you rather just stick with the simplicity of the previous skill tree?

- Not too much, I think this is great! If anything, I would say the existing skill-tree is too little :+1: . The SFL core game is great, but it definitely needs a lot more content. This proposal is fantastic.

Is the UI an issue? Does having to scroll down a bit to see the last tier of skills cause annoyances?

- I really like [this UI](https://user-images.githubusercontent.com/108110065/216873632-b7823b4c-48d9-435a-8e85-3ac9d13a248c.png) you posted. I think [this other UI](https://user-images.githubusercontent.com/108110065/216869616-2d8e2e34-f930-48ff-9f9a-b8fd6c26f556.png) looks acceptable too - maybe just a few sizing tweaks here or there. I don't really mind scrolling. I think what's most important is that the easier skills to unlock are the first ones you see, as this caters to new players and won't overwhelm them. I don't think seasoned players will care as much.

Do you like coffee?

Yup!

Final note - I think this proposal is indirectly coupled to the [multiple bumpkins proposal](https://github.com/sunflower-land/sunflower-land/discussions/2211). I think getting an overhaul'd skill-tree like this will help make multiple bumpkins make more sense.

**Reply by TheDcol** [reply]
Thanks Minion!

I can also see how the multiple bumpkin proposal can work in tandem with revamped skills... It will need a GOOD look at the balancing aspect of it though.. which I am happy to spend time on if needed 😅

**Reply by rodolfoddias** [reply]
I think it depends on which behavior we are aiming to incentivize. If we allow players to combine multiple bumpkins skills we might not have a single Bumpkin at max level, for example. 

I would stick with Main Bumpkin skills and use different Bumpkin to equip different pets, wearables and stuff like that. I think it would be better to resource burning if we force players to use all their resources into a single character (since the leveling requirements scale). It would also keep the Multiple Bumpkins proposal way more simple. 

Overall, I really loved the skill revamp proposal @TheDcol. It would definitely give me extra reasons to grind - nowadays I am barelly cooking.

**Reply by TheDcol** [reply]
Thanks @rodolfoddias ,

The multi bumpkin needs to be carefully looked at for sure.. One way to incentivise players to level up all bumpkins BEYOPND level 20 would be to also have skills be level locked. So if you want to unlock a Red skill, your bumpkin needs to be level 30 or 50... what do you think?

^This would also incentivise players to level up all their bumpkins even more, which is more burn created

**Reply by rodolfoddias** [reply]
That's a solution, but we would add another layer of complexity to the development and I am not sure it would add a great value to the game itself to justify it. If players choose to level up multiple Bumpkins, it will probably be because it is easier to obtain more skill levels by leveling multiple Bumpkins to a certain level. Is that what we want?

When I think about what is the best to the economy, I would choose whatever takes more time / consumes more resources and that is easier to code. This way the team would have more time for new releases and more resources would be burned. Developing a system that considers multiple Bumpkin Skills is certainly more complex than developing a system that considers a single Bumpkin skills. In order to release it faster, I would focus on one Bumpkin. 

Complexity is good, but it should be added only where it brings real value. Otherwise it will just make feature releases slower.

**Reply by TheDcol** [reply]
Yup, so level locking skills is something Craig agrees with too having spoken with him (since ge us working on multiple bumpkins, kind of have to work together to an extent)

It would use more resources and be overall good for the economy IMO, and as far as coding goes Adam said adding restraints is fine

Personally, I would prefer to stick with 1 bumpkin related to skills only as that is how I originally designed the idea but if devs want to go ahead with multi bumpkin, it can be changed

**Comment by Rook982** [comment]
Loved this post, the current skill tree is too generic and needs more complexity especially if we want to make SFL a long term game for everyone. I love specializations, why stick to 23 max points for a tree, make it 30 or 40 even with absurd final skills! Introduce hybrid systems and a jack of all trades build. 

With the plan for multiple bumpkins that will be put into a role type system of who does what. It will still be possible to have it all but you are gonna have to work hard to get it to that level. 

Is it too much? Would you rather just stick with the simplicity of the previous skill tree?
        No previous skill tree is too simple and is lopsided in what it gives out. 
        
Is the UI an issue? Does having to scroll down a bit to see the last tier of skills cause annoyances?
        UI is fine for me with phones these days, everything can support the scrolling down of trees, just coordinate with artists on having unique and recognizable icons and writers with **clear skill explanations**. If you can even include the actual percentage chance of a skill occuring that would be even better instead of today's "Chance" description. Theory crafting is rabbit hole gamers often fall into. Having exact numbers for a skill helps a players choose what he wants instead of a hopeful "this can be good for my farm". 
        
Plus with the recent post of people wanting a rebalance of land resource distribution, I suggested that your Bumpkin specialization could affect your land plots. So more skills points into a tree could provide a modifier in adding a specific resource/plot on your farm. You could even add a skill that straight up lets you add a resource node to your farm. i.e. 2-3 point skill, Iron Miner's instinct: Add 1 Iron node to your farm. Can be farm plot, stone or even gold as long as you progress far enough into the skill tree. Removes a portion of RNG in the land resource distribution and again reinforces specialization.

**Reply by TheDcol** [reply]
Thank you :)

>why stick to 23 max points for a tree, make it 30 or 40 even with absurd final skills

The option would be to add branches or keep the tree going further depending on new features being added, so the trees could expand.

Skills related to expansions would be cool and I did think about it, but I decided to exclude as a lot of players have already expanded 15 times.. unless SFL was to restart for everyone mm..

**Reply by Rook982** [reply]
Yeah, it would have to restart indeed but if they do, they would now need to standardize the plots/resources we get every expansion. Now players can truly decide on a specialization and removes RNG from land and becomes an RPG of sorts.

**Comment by ghost** [comment]
More skills will allow you to pump your Bumpkins more efficiently when several Bumpkins are added. But I think we need to balance the skill points so that there is no need to raise the levels to 100+

**Reply by TheDcol** [reply]
Thanks for the comment mate, what do you mean by 'no need to raise levels to 100+'? Are you saying that all skills should be obtainable at a lower level?

**Reply by ghost** [reply]
As far as I understand, they want to add up to 3 Bumpkins to the farm, this can be used to avoid raising levels. You can also get skill points at each level. This way we will get 3*40= 120 skill points. And you can already work with this.

**Reply by TheDcol** [reply]
I think the goal for revamped skills is to require more time and effort in acquiring many skills, adding a sort of 'end goal' component which currently severly lacks

**Reply by ghost** [reply]
Well , I think it 's worth combining this with the fact that there will be several bumpkins .

**Reply by TheDcol** [reply]
Yes, it has been stated that this proposal and the multi bumpkin proposal need to go hand-in-hand

The current thought is that you can level up each bumpkin on the farm and each bumpkin selects their own skills.

Skills will be level locked along with making it costly to have 3-4 bumpkins on the island at once. All bumpkins wearables active.

Will need to be balanced accordingly

**Reply by ghost** [reply]
I must have misunderstood you.

**Reply by TheDcol** [reply]
All good mate

**Comment by brian-man** [comment]
I gave @TheDcol feedback about this privately, but I'll post it here as well.  This comment is not about a specific tree or box but about a general concern to keep in mind when designing.

A -> B -> C
or
D -> E -> F

With such a shape if I only want C or F (out of all 6) then I'm "stuck" with A+B or D+E.  Sometimes that's intentional by the devs, sometimes not as part of the skill decisions.

If I don't want B or E, then it kind of sucks.

-----

For the 1-2-1 trees, it's a similar problem but slightly better.  Best, IMO, would be 2-2-2 trees where you can pick either from the first bucket and either from the second bucket and either from the third bucket.

Example:
(A or B) -> (C or D) -> (E or F)

Here I can choose among:
A
B
AC
AD
BC
BD
ACE
ACF
ADE
ADF
BCE
BCF
BDE
BDF

**Reply by TheDcol** [reply]
Appreciate it as always mate <3

The one issue I came across when thinking about the design you proposed was how multiple bumpkins (which will be a thing) would impact it and if players could then unlock more skills more quickly than intended. 

Personally, I am designing this in a way in which high-end skills are REALLY good and if you want to unlock them, you must unlock the WHOLE tree. The premise is that lvl 30+ players are 'bored' with nothing to really aim for currently, and this would provide part of a solution to them trying to unlock that really good skill.

I think it's also about making the paths 'make sense' and entice the player that picks skill A, to pick skill B, skills that do well together be on the same side of the branch

I am not againt a player having 100% freedom to choose whatever skills they want and skip the ones they do not like, but I am worried at the effects of giving TOO much freedom. Cost/drawbacks are something really lacking in this game.

To be honest, I also have not had much time to dive too deep into the other trees, I tried a cascading tree but was not liking it personally and I decided to devote some of my own free time to develop this further (along with like 5 other proposals ongoing/in discussion.. plus actual work during the week)

I think you or someone should look at building out the style of tree you mentioned in more detail and think of all the issues/drawbacks/possibilities that could spawn from it :)

I am posting an updated tree below soon, again, just ideas :D

**Comment by RonaldKoeman6356** [comment]
This is so crazy

**Comment by TheDcol** [comment]
Updated skill trees, have not spent a lot more time on this as I am waiting on multi bumpkin proposal to be finalised so we have a better understanding of how that system would flow with skills.

168 skill points required CURRENTLY, so let's say 1 skill point every 2 levels.. 336 levels required to unlock them all :D
And more skill trees to come with new features.. So that would be quite a bit for players to progress to if they wish.

Farmer I II and III+ would be the new Scarecrow sfts from helios, renamed to Farmer x. Also anything with I II etc. would be sfts.
Cows, diamonds, mutant crops and other skills that you might not have a clue about are all features currently being developed.

**VALUES/SKILLS NOT FINAL**

![image](https://user-images.githubusercontent.com/108110065/220784770-f3a9ff5f-58ba-482f-8d14-cd450eabf83c.png)
![image](https://user-images.githubusercontent.com/108110065/220784725-e591f0f6-d021-47e1-b10d-559c74c0622b.png)
![image](https://user-images.githubusercontent.com/108110065/220784737-71a5ef30-9f36-4e2d-8f99-70fd475f9885.png)
![image](https://user-images.githubusercontent.com/108110065/220784748-3ae42ea2-77c7-4cc6-90eb-bc0aeb2613a4.png)

Below is an example of locking skills to level requirements, this was discussed in the multi-bumpkin proposals where players would have 2-3 bumpkins on a farm and have all their skills/wearables active.

This would be to ecnourage players to level up all their bumpkins to get a lot more skills across or focus on 1 bumpkin to fully unlock a tree. VALUES ARE DRAFT

![image](https://user-images.githubusercontent.com/108110065/220784991-30ab9702-baa2-4221-bf33-9e7103b50956.png)

**Reply by Xeko-92** [reply]
I know the skills aren't final obviously, but when would you like input on them?

**Reply by TheDcol** [reply]
On the actual skills themselves? Whenever 😁

Adam posted this, it is the likely direction they will go https://github.com/sunflower-land/sunflower-land/discussions/2180#discussioncomment-5084747

**Comment by DeshFarm** [comment]
Acho bacana as ideias de skills mas acho que seria bom reavaliar algumas skills propostas pois irá faltas pontos para isso tudo, as skills que temos hoje é algo único espero que não perca a essência das skills atuais com essas novas propostas.

I think the skills ideas are cool but I think it would be good to reassess some of the proposed skills because points will be missing for all of this, the skills we have today are something unique, I hope you don't lose the essence of the current skills with these new proposals.

**Reply by TheDcol** [reply]
Hi mate thanks for the comment, what do you mean by 'points will be missing for all of this'?

There will be more levels/more skill points in the game, if that is what you mean.

> I hope you don't lose the essence of the current skills with these new proposals'

Majority of the current skills are present in here too, current skills are too limiting

**Comment by adamhannigan** [comment]
@TheDcol Incredible work - I will relay a few notes we've discussed 🤗 

I've gone through the latest addition of skills and made some notes.

There are some skills that will provide some complex scenarios that could be abused - these are skills that give you one-off access to items such as (Farms, Fruit SFTs etc.). While there are solutions, I feel these will lead to a complex system with more avenues for abuse.

The direction we want to take is to provide new gameplay opportunities unlocked through skills. E.g. Feed animals with Kale, Golden Axe, New Fruit Seeds + New Crop Seeds. However, I believe we will take too long if we build out this custom skills.

I would suggest that we revamp the basic tiers and then slowly introduce the tier 4 skills which will require more game design + dev work.

## Crops

**1 skill point**
- 5% faster growth
- 5% chance for +1 drop
- +0.1 yield Sunflowers, Potatoes, Pumpkins + Carrots)

**2 skill points**
- +0.15 yield Cabbage, Beetroots, Cauliflowers & Parsnips
- +0.2 yield Radish Wheat + Kale

**3 skill points**
- Wheat grows 20% faster
- High demand - +10% profit selling crops
- Composter +0.05

**Future ones we can design = 4 skill points**
- Double sprinkler effect
- Unlock Exotic Seeds
- Golden Flowers
- +6 seed bonus
- Unlock more crops (coffee, tomato, others)
- Farmers

## Fruit

**Tier 1**
- Fruit grows 5% faster
- +0.1 Berry Yield
- +0.1 Orange and Apple Yield

**Tier 2**
- Fruit worth 10% more
- Berries grow 10% faster
- Apples & Oranges grow 10% faster

**Tier 3**

- Wheel barrow +0.05

**Future**
- New fruit seeds

## Trees

**Tier 1**
- Trees Drop +0.1
- Trees grow 20% quicker

**Tier 2**
- Drop Acorns = Roasted Acorns Food. This is a new feature, but lightweight on dev since we already have drops + cooking.
- Critical Drop (Roughly an extra +0.1)

**Tier 3**
- Insta grow (10% of time)
- Lumberjack +0.1

**Future**
- Wood  in cooking
- Fairies

## Animals

**Tier 1**
- +0.1 milk yield  (Coming soon)
- +0.1 egg yield
- 10% cheaper animals

**Tier 2**
- Cows 10% faster  (Coming soon)
- Chickens 10% faster
- Egg Critical Drop

**Tier 3**
- +5 Capacity Hen House
- +4 Capacity Cow Barn (Coming soon)

**Coming Soon**
- Pets stay happy twice as long  (Coming soon)

## Rocks

**Tier 1**
- +0.1 Stone
- +0.1 Iron

**Tier 2**
- Stone recovers 20% faster
- Iron recovers 20% faster

**Tier 3**
- +0.1 Gold Drops
- Gold recovers 20% faster
- Quarry +0.1

**Future**
- Boulder boosts
- Quarry Boosts

## Food

**Tier 1**
- Meals cook 10% faster
- Bonus 5% SFL from Grub Shop

**Tier 2**
- Bonus 10% SFL from cakes
- Bonus 10% experience from Juices
- Bonus 5% experience from edibles

**Tier 3**
- 10% Bonus SFL from Deli Foods
- Cook Cakes 20% faster

**Future**
- Farmers/Chefs
- Resource drops
- Wood in cooking

**Reply by TheDcol** [reply]
Looks solid to me, and makes sense 👍

**Reply by ghost** [reply]
Will the skills only swing on one bumpkin?
I thought the point was to make more bumpkins and make it harder to upgrade skills by increasing them

**Reply by TheDcol** [reply]
I replied to one of your other comments above

**Reply by JohnnConstantine** [reply]
WWhat TheDcol proposes is much more interesting and gives an objective to follow.
You should make it so that to reach the last talent of a specialization you have to use all the points in that same branch. So when we have 4 Bumpkins we must make them all reach level 100 if we want to have all the talents of all the branches.

This allows strategies to be formed. You can decide to level up all Bumpkins in a similar way or you can decide to focus on leveling up 1 specific Bumpkin to enable the "op skill" of the branch you focus on.

Clarification: 4 Bumpkins is an example, ideally we can have 1 Bumpkin for each branch of specialization. Either 4 or 10.
And getting to level 100 must take a lot of time and resources.

**Comment by alphacruze** [comment]
Hmm, what about making the "final" skill something like an "active" skill

As right now, everything is just "passive", this could be added onto the idea of having the requirements of certain things being done to unlock certain skills.

The idea is that once someone has reached the very last "skill" or as I call it "ultimate skill" within a skill tree, they will get a "Active" skill
These active skills can have different effects for example:

The ultimate skill in the Crop farming category could be something like:

"All planted crops are instantly grown and harvested"
With a cooldown of maybe 24 hours, or resets at a the changing of the day.

I gotta think... about the rest

**Reply by adamhannigan** [reply]
I like this idea ❤️ Lots of games (especially fighting ones) have these mechanisms.

**Reply by adamhannigan** [reply]
Building on top of your idea:

Crops: Instant Harvest 5 crops
Fruit: Double Fruit Drop for 3 trees
Trees: Chop down 3 trees without axes
Stones: Double Stone Drop for 3 rocks
Cooking: Instant Cook 1 recipe
Animals: Feed 3 chickens for free

**Reply by brian-man** [reply]
> Fruit: Double Fruit Drop for 3 trees
> Stones: Double Stone Drop for 3 rocks

These are likely to trigger a ton of "it's not working" discussions in farmers-chat because of the "applies to next growth/replenish/regen" design of the game.

**Reply by TheDcol** [reply]
Opens a lot of doors and opportunities, something to think about!

**Reply by alphacruze** [reply]
It definitely opens a lot of doors

And it's not even a necessity for it to be the "ultimate".
After all it's possible to make it another part of the skill tree, like choose either a "passive"

As an example the choices would go 

"+10% crop yield"
Or 
Only possible when crops are ready to harvest
"Harvest 2 crops double yield"

So for every skill, just make a Active skill as an option, it will still follow the same tree, it just 2 options instead of one

The tree itself wouldn't change, it's just there's more options for skills and variations without needing to making the trees 3/4 lines

This could be done as a popup once someone wants to choose the skill in a tree, they would get a pop-up which has them choosing, passive or active skill.

All Active skills will naturally have a cooldown, after all it's "active"
And with this idea, people can choose to have a combination of actives and passives, or just actives/just passives
Many new playstyles would thus be born

**Comment by TCoutz** [comment]
I really liked everything. I just ask to please have some sort of combat/exploration gameplay in the future. I really like exploration in games like Stardew valley. It would be really great to have a combat skill tree in the future. Fishing and bees too please 🥺

**Reply by TheDcol** [reply]
Would be cool!

**Comment by Ivan-Ricardo-Lopes** [comment]
What happens with the legacy skills from beta? I am asking because I just bought a land with legacy skills and it was expensive, Will I lose it?

**Reply by TheDcol** [reply]
Those legacy skills will remain AFAIK

**Comment by Chucrutty** [comment]
[Github_SFL_proposta ENG.pptx](https://github.com/sunflower-land/sunflower-land/files/10830563/Github_SFL_proposta.ENG.pptx)
Dcol, I made a draft of my idea that can have cynergy with your idea of ​​skills

maybe you can digest the idea
